e2e/content-services/search/search_filters.e2e.ts

```diff
@@ -100,4 +119,24 @@ describe('Search Filters', () => {
         searchFiltersPage.clickSizeShowLessButtonUntilIsNotDisplayed();
     });
 
+    it('[C287796] Should be able to display the correct bucket number after selecting a filter',  () => {
+        browser.get(TestConfig.adf.url + '/search;q=*');
+
+        searchFiltersPage.clickPngImageType();
+
+        let bucketNumberForFilter = searchFiltersPage.getBucketNumberOfFilterType(filter.type, filter.name.length);
+
+        let resultFileNames = contentList.getAllRowsNameColumn();
+
+        expect(bucketNumberForFilter).not.toEqual('0');
+
+        expect(paginationPage.getTotalNumberOfFiles()).toEqual(bucketNumberForFilter);
+
+        resultFileNames.then((fileNames) => {
```

Comment:
I think in this case this line is not necessary. I think it will work like this: 
resultFileNames.map((nameOfResultFiles) => {
                expect(nameOfResultFiles).toContain('.png');
            });

Reply:
I tried this method but it's not working like that.

Reply:
Ok, then we can leave it as it is for now. Probably it's not working cause we do a defer fulfil in contentList.getAllRowsNameColumn().